tests/aurora/ndsrom.cpp

```diff
@@ -0,0 +1,182 @@
+/* xoreos - A reimplementation of BioWare's Aurora engine
+ *
+ * xoreos is the legal property of its developers, whose names
+ * can be found in the AUTHORS file distributed with this source
+ * distribution.
+ *
+ * xoreos is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ *
+ * xoreos is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with xoreos. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file
+ *  Unit tests for our NDS ROM file archive class.
+ */
+
+#include "gtest/gtest.h"
+
+#include "src/common/error.h"
+#include "src/common/memreadstream.h"
+
+#include "src/aurora/ndsrom.h"
+
+// Percy Bysshe Shelley's "Ozymandias"
+static const char *kFileData =
+	"I met a traveller from an antique land\n"
+	"Who said: Two vast and trunkless legs of stone\n"
+	"Stand in the desert. Near them, on the sand,\n"
+	"Half sunk, a shattered visage lies, whose frown,\n"
+	"And wrinkled lip, and sneer of cold command,\n"
+	"Tell that its sculptor well those passions read\n"
+	"Which yet survive, stamped on these lifeless things,\n"
+	"The hand that mocked them and the heart that fed:\n"
+	"And on the pedestal these words appear:\n"
+	"'My name is Ozymandias, king of kings:\n"
+	"Look on my works, ye Mighty, and despair!'\n"
+	"Nothing beside remains. Round the decay\n"
+	"Of that colossal wreck, boundless and bare\n"
+	"The lone and level sands stretch far away.";
+
+/* Percy Bysshe Shelley's "Ozymandias", within a NDS ROM file.
+ * Note: this isn't really a valid, standard NDS file. It only
+ * fills the header fields we actually care about. If the NDSFile
+ * class is extended to care about the other fields as well, this
+ * file data has to be changed as well. */
```

Comment:
Did you manually generate this? I think the test would be better if generated from some tool.